### DISCUSSION TITLE
Need to generate single war file to deploy in tomcat

### DISCUSSION THREAD

**Discussion by ravidevops123** [shopizer-ecommerce/shopizer#675] [Q&A]
Hi this is ravikumar

I need this application to run in tomcat application server
So for this need to generate single war file and need to deploy in tomcat app server
can you please suggest me

**Comment by stevanusc6** [comment]
Maybe you can see how spring boot deployed on tomcat in here https://www.baeldung.com/spring-boot-war-tomcat-deploy

**Comment by shopizer-ecommerce** [comment]
Hi Ravikumar

For running thin is tomcat you need to use shopizer 2.X (latest 2 version) then run mave clean install to run a war file that can be deployed to tomcat

**Reply by ravidevops123** [reply]
Still iam able to generate war file please give suggestions
